tensorflow_addons/callbacks/tqdm_progress_bar.py

```diff
@@ -0,0 +1,190 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""TQDM Progress Bar. """
+
+from __future__ import absolute_import, division, print_function
+
+import time
+from collections import defaultdict
+
+from tensorflow.keras.callbacks import Callback
+from tensorflow_addons.utils import keras_utils
+from tqdm.auto import tqdm
```

Comment:
Since tqdm isn't a requirement for addons could you put this import as a try-catch within the class that returns an error if tqdm is unable to import

Reply:
got it, will do!

Reply:
updated in latest patch!